oxidation/bindings/android/libparsec/rust/src/lib.rs

```diff
@@ -27,7 +27,7 @@ pub extern "C" fn Java_com_scille_libparsec_Runtime_startRuntime(
 
 /// Interface between Web, Rust and Android
 /// Cordova bridge API for Android requests the inputs and output to be passed as a JS Object
-/// hence we have to comply with this ourself to provide the same API accross plateforms
+/// hence we have to comply with this ourself to provide the same API across plateformes
```

Comment:
no `e` needed here see (https://www.britannica.com/dictionary/platform)

> The program can be run on various PC platforms.

Reply:
It's platforms without any `e`, `plateformes` is the correct word in french but the text here is in english

Reply:
done